src/Components/ArticleMainCard.jsx

```diff
@@ -34,8 +54,9 @@ function ArticleMainCard() {
 
       <p>{currentArticle.body}</p>
       <p>
-        <button>👍</button> {currentArticle.votes}
+        <button onClick={handleClick}>👍</button> {currentArticle.votes}
       </p>
+      {error ? ( <p>Refresh the page</p>):null}
```

Comment:
It would be good to have a more descriptive error message - this doesn't explicitly say there is an error so it is not clear why a user needs to refresh the page.

Reply:
After change  that can I merge straight away or I need to submit a new PR?? Let me know please 

> It would be good to have a more descriptive error message - this doesn't explicitly say there is an error so it is not clear why a user needs to refresh the page.